core/CHANGELOG.T2T1.md

```
@@ -1331,6 +1348,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 [#6370]: https://github.com/trezor/trezor-firmware/pull/6370
 [#6394]: https://github.com/trezor/trezor-firmware/pull/6394
 [#6435]: https://github.com/trezor/trezor-firmware/pull/6435
+[#6442]: https://github.com/trezor/trezor-firmware/pull/6442
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "`#6442`"

(MD053, link-image-reference-definitions)